src/app-layout/visual-refresh/context.tsx

```diff
@@ -418,7 +362,7 @@ export const AppLayoutInternalsProvider = React.forwardRef(
       (!toolsHide && isToolsOpen) ||
       (splitPanelDisplayed && splitPanelPosition === 'side' && isSplitPanelOpen);
     const hasDrawerViewportOverlay =
-      isMobile && (!!activeDrawerId || (!navigationHide && isNavigationOpen) || (!toolsHide && isToolsOpen));
+      isMobile && (!!activeDrawerId || (!navigationHide && navigationOpen) || (!toolsHide && isToolsOpen));
```

Comment:
Before `isNavigationOpen` was a processed value with controllability applied. Now we can trust `navigationOpen`, so there is a big find-and-replace `isNavigationOpen` -> `navigationOpen` across this component